.github/actions/issue_closer/action.yml

```diff
@@ -0,0 +1,9 @@
+name: 'Issue Closer'
+description: 'Close GitHub Issues on PR merges to non-default branches'
+inputs:
+  token:
+    description: 'A GitHub token with PR read and Issue close permissions'
+    required: true
+runs:
+  using: 'node16'
```

Comment:
Why did you choose to use Node16?

Reply:
I think that came from an official GitHub doc on how to create a JavaScript action. Everything was tested locally using v20. We can try to upgrade the next time dev work (including all of the testing setup) is done on the action.